.github/workflows/compile-artifact.yml

```diff
@@ -35,7 +41,8 @@ jobs:
       - name: Configure bundler
         run: |
           mkdir -p vendor/bundle
-          bundle config path vendor/bundle
+          bundle config set --local path vendor/bundle
+          bundle config set --local gemfile Gemfile${{ inputs.gemfile_suffix }}
 
       - name: bundle cache
```

Comment:
how do you handle the github bundle cache key?  In the original flows the cache key was a hash of the gemfile.foo.lock contents..
this way when any gem version changed, the cache was invalidated.

Reply:
oh! very good point...

... Might need to bug you for ideas about this on Slack.

Reply:
I put in your fix, but we may need to adjust the env vars as you mentioned below for it to work :grimacing:

Reply:
After a Zoom call, we determined that the local variables are the highest precedence and Buddy was set to use those, so it's best to continue with what we had (and with what also has most precedence)

Reply:
Modified to not need the env var. Should work now!